src/components/Person/PersonLink.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person } from '../../types';
+
+interface Props {
+  person: Person,
+}
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const {
+    name, sex, born, died, fatherName, motherName, slug, mother, father,
+  } = person;
+
+  const { personSlug } = useParams();
+  const selectedUser = personSlug;
```

Comment:
```suggestion
  const { personSlug: selectedUser } = useParams();
```
selectedUser seems to be a redundant variable. You can rename variables (from object) when doing a destructuring assignment